### ISSUE TITLE
Site not correctly deployed to s3

### ISSUE THREAD

**Issue by danielfdsilva** [openaq/openaq-metadata-editor#48]
S3 by default does not work with SPAs. When an url is loaded it interprets the path segments as folders and throws an error because the file was not found.
Ex: https://metadata.openaq-staging.org/location/AE-2

This is fixable by redirecting the requests to the index document. Here's a [guide](https://medium.com/@P_Lessing/single-page-apps-on-aws-part-1-hosting-a-website-on-s3-3c9871f126) that may help.

**Comment by danielfdsilva** [comment]
cc @jflasher

**Comment by olafveerman** [comment]
@jflasher I think this has to be set up through the interface. Let us know if this is something we can solve on our end.

**Comment by sethvincent** [comment]
I think all we need is to set a custom error response in the cloudformation distribution, for example:

```
CustomErrorResponses:
  - ErrorCode: 404
    ResponseCode: 200
    ResponsePagePath: /index.html
```

**Comment by jflasher** [comment]
This is deploying now, usually takes ~10 minutes to fully deploy.

<img width="1018" alt="Screen Shot 2019-07-24 at 10 48 47 AM" src="https://user-images.githubusercontent.com/848934/61803738-b74caf80-ae00-11e9-8c19-a992e0c7c9df.png">

**Comment by sethvincent** [comment]
Still seeing access denied errors. I wonder if we also need to revise the bucket policy to be more permissive.

**Comment by jflasher** [comment]
Should this work the same way as the main website? I can ensure S3 and CF are set up the same if so.

**Comment by sethvincent** [comment]
@jflasher yes, I believe so.

**Comment by jflasher** [comment]
I turned on the S3 website hosting feature and can confirm that a url like http://metadata.openaq-staging.org.s3-website-us-east-1.amazonaws.com/location/AE-2 works as expect. 

I updated the CloudFront origin to be this endpoint, which matches the behavior of openaq.org. I think this will fix it once it propagates through CF? 

I have removed the error page redirect in CF as I think that ended up being not needed. openaq.org CF also has separate paths for *.js and *.css but I do not believe they are actually doing anything. If you notice anything weird with js/css, let me know and I can mimic the openaq.org behavior.
